src/components/CopyableConversionInput.tsx

```diff
@@ -57,7 +57,7 @@ function InputCopy({ name, useState, ...props }: InputCopyProps): ReactElement {
                 .value;
             else throw new Error('Copyable ConversionInput must have a name');
             if (inputValue) {
-              await navigator.clipboard.writeText(inputValue);
+              void window.navigator.clipboard.writeText(inputValue);
```

Comment:
why this change?

Reply:
This operation is async; await is meaningless in this context, we want to just call something without the need to get the result, thus void. Window for the code transparency.